lib/isgeocsv.js

```diff
@@ -0,0 +1,20 @@
+var geocsvinfo = require('geocsv-info');
```

Comment:
Since we're only using two smaller functions from `geocsv-info`, maybe we can just implement them in mapbox-file-sniff? Though that would entail passing over the tests into mapbox-file-sniff as well.